lib/pipelineTemplateFactory.js

```diff
@@ -14,6 +14,21 @@ class PipelineTemplateFactory extends TemplateMetaFactory {
         return 'PIPELINE';
     }
 
+    /**
+     * List pipeline templates
+     * @method list
+     * @param  {Object}   config                  Config object
+     * @param  {Object}   config.params           Parameters to filter on
+     * @return {Promise}                          Resolve pipeline template meta
+     */
+    async list(config) {
+        if (config.params) {
+            config.params.templateType = this._getTemplateType();
+        }
+
+        return super.list(config);
+    }
+
```

Comment:
This is not needed. This should be abstracted into parent class `TemplateMetaFactory`
```suggestion
```